.vscode/settings.json

```diff
@@ -52,10 +51,10 @@
     "editor.tabSize": 2,
     "editor.defaultFormatter": "vscode.json-language-features",
   },
-  "typescript.tsdk": "node_modules/typescript/lib",
   "files.insertFinalNewline": true,
   "clang-format.language.typescript.enable": false,
   "editor.rulers": [
     180
-  ], // ESLint `max-len` rule.
+  ],
+  "typescript.tsdk": "node_modules\\typescript\\lib", // ESLint `max-len` rule.
```

Comment:
Well I guess the extension forcefully writes where it found it... So much for removing it.